docs/ibc/Slashing.md

```diff
@@ -0,0 +1,7 @@
+# Slashing Evidence Handling
```

Comment:
IMO slashing may have to be it's own section?

There is a lot to discuss with regards to this, and different types of slashing that could be implemented in the future.